### PR TITLE
uwuified everything

### PR DESCRIPTION
**Pull request by olibols**
-made uwus and owos allowed. dont mark as invalid pls i need it for hacktoberfest ily uwu

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,83 +1,83 @@
...
 
-## Attribution
+## Attwibution
 
-This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 1.4,
-available at https://www.contributor-covenant.org/version/1/4/code-of-conduct.html
+This Code of Conduct is adapted fwom the [Contwibutow Cuvnyant][homepage], vewsion 1.4,
+avaiwabwe at https://www.contwibutow-cuvnyant.owg/vewsion/1/4/code-of-conduct.htmw
 
-[homepage]: https://www.contributor-covenant.org
```
Comment (Cerbiac) [inline CODE_OF_CONDUCT.md]: Broken Link

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -1,83 +1,83 @@
...
-members of the project's leadership.
+Pwoject maintainyews who do nyot fowwow ow enfowce the Code of Conduct in good
+faith may face tempowawy ow pewmanyent wepewcussions as detewminyed by othew
+membews of the pwoject's weadewship.
 
-## Attribution
+## Attwibution
 
-This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 1.4,
-available at https://www.contributor-covenant.org/version/1/4/code-of-conduct.html
```
Comment (Cerbiac) [inline CODE_OF_CONDUCT.md]: Broken Link

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -1,83 +1,83 @@
...
+This Code of Conduct is adapted fwom the [Contwibutow Cuvnyant][homepage], vewsion 1.4,
+avaiwabwe at https://www.contwibutow-cuvnyant.owg/vewsion/1/4/code-of-conduct.htmw
 
-[homepage]: https://www.contributor-covenant.org
+[homepage]: https://www.contwibutow-cuvnyant.owg
 
-For answers to common questions about this code of conduct, see
-https://www.contributor-covenant.org/faq
```
Comment (Cerbiac) [inline CODE_OF_CONDUCT.md]: Broken Link

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -1,83 +1,83 @@
...
+wepwesenting a pwoject ow communyity incwude using an officiaw pwoject e-maiw
+addwess, posting via an officiaw sociaw media account, ow acting as an appointed
+wepwesentative at an onwinye ow offwinye event. Wepwesentation of a pwoject may be
+fuwthew definyed and cwawified by pwoject maintainyews.
 
-## Enforcement
+## Enfowcement
 
-Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported by contacting the project team at complaints@leijurv.com. All
```
Comment (Cerbiac) [inline CODE_OF_CONDUCT.md]: Broken Email

File: FEATURES.md [hunk 5]
```diff
@@ -1,52 +1,52 @@
...
-- **Chunk caching** Baritone simplifies chunks to a compacted internal 2-bit representation (AIR, SOLID, WATER, AVOID) and stores them in RAM for better very-long-distance pathing. There is also an option to save these cached chunks to disk. <a href="https://www.youtube.com/watch?v=dyfYKSubhdc">Example</a>
-- **Block breaking** Baritone considers breaking blocks as part of its path. It also takes into account your current tool set and hot bar. For example, if you have a Eff V diamond pick, it may choose to mine through a stone barrier, while if you only had a wood pick it might be faster to climb over it.
-- **Block placing** Baritone considers placing blocks as part of its path. This includes sneak-back-placing, pillaring, etc. It has a configurable penalty of placing a block (set to 1 second by default), to conserve its resources. The list of acceptable throwaway blocks is also configurable, and is cobble, dirt, or netherrack by default. <a href="https://www.youtube.com/watch?v=F6FbI1L9UmU">Example</a>
```
Comment (Cerbiac) [inline FEATURES.md]: Broken Link

File: FEATURES.md [hunk 6]
```diff
@@ -1,52 +1,52 @@
-# Pathing features
-- **Long distance pathing and splicing** Baritone calculates paths in segments, and precalculates the next segment when the current one is about to end, so that it's moving towards the goal at all times.
-- **Chunk caching** Baritone simplifies chunks to a compacted internal 2-bit representation (AIR, SOLID, WATER, AVOID) and stores them in RAM for better very-long-distance pathing. There is also an option to save these cached chunks to disk. <a href="https://www.youtube.com/watch?v=dyfYKSubhdc">Example</a>
```
Comment (Cerbiac) [inline FEATURES.md]: Broken Link

File: CODE_OF_CONDUCT.md [hunk 7]
```diff
@@ -1,83 +1,83 @@
...
 
-## Our Standards
+## Ouw Standawds
 
-Examples of behavior that contributes to creating a positive environment
-include:
+Exampwes of behaviow that contwibutes to cweating a positive enviwonment
+incwude:
 
-* No Anime (including uwu's or owo's)
```
Comment (Cerbiac) [inline CODE_OF_CONDUCT.md]: OwO's are now allowed.

File: FEATURES.md [hunk 8]
```diff
@@ -1,52 +1,52 @@
...
-- **Minimum improvement repropagation** The pathfinder ignores alternate routes that provide minimal improvements (less than 0.01 ticks of improvement), because the calculation cost of repropagating this to all connected nodes is much higher than the half-millisecond path time improvement it would get.
-- **Backtrack cost favoring** While calculating the next segment, Baritone favors backtracking its current segment. The cost is decreased heavily, but is still positive (this won't cause it to backtrack if it doesn't need to). This allows it to splice and jump onto the next segment as early as possible, if the next segment begins with a backtrack of the current one. <a href="https://www.youtube.com/watch?v=CGiMcb8-99Y">Example</a>
```
Comment (Cerbiac) [inline FEATURES.md]: Broken Link

File: FEATURES.md [hunk 9]
```diff
@@ -1,52 +1,52 @@
...
-- **Incremental cost backoff** When path calculation exits early without getting all the way to the goal, Baritone it needs to select a segment to execute first (assuming it will calculate the next segment at the end of this one). It uses incremental cost backoff to select the best node by varying metrics, then paths to that node. This is unchanged from MineBot and I made a <a href="https://docs.google.com/document/d/1WVHHXKXFdCR1Oz__KtK8sFqyvSwJN_H4lftkHFgmzlc/edit">write-up</a> that still applies. In essence, it keeps track of the best node by various increasing coefficients, then picks the node with the least coefficient that goes at least 5 blocks from the starting position.
```
Comment (Cerbiac) [inline FEATURES.md]: ~~Broken Link~~ Sike

File: FEATURES.md [hunk 10]
```diff
@@ -1,52 +1,52 @@
...
+- **Backtwack cost favowing** Whiwe cawcuwating the nyext segment, Bawitonye favows backtwacking its cuwwent segment. The cost is decweased heaviwy, but is stiww positive (this won't cause it to backtwack if it doesn't nyeed to). This awwows it to spwice and jump onto the nyext segment as eawwy as possibwe, if the nyext segment begins with a backtwack of the cuwwent onye. <a hwef="https://www.youtube.com/watch?v=CGiMcb8-99Y">Exampwe</a>
+- **Backtwack detection and pausing** Whiwe path cawcuwation happens on a sepawate thwead, the main game thwead has access to the watest nyode considewed, and the best path so faw (those awe wendewed wight bwue and dawk bwue wespectivewy). When the cuwwent best path (wendewed dawk bwue) passes thwough the pwayew's cuwwent position on the cuwwent path segment, path execution is paused (if it's safe to do so), because thewe's nyo point continyuing fowwawd if we'we about to tuwn awound and go back that same way. Nyote that the cuwwent best path as wepowted by the path cawcuwation thwead takes into account the incwementaw cost backoff system, so it's accuwate to what the path cawcuwation thwead wiww actuawwy pick once it finyishes.
 
-# Chat control
```
Comment (Cerbiac) [inline FEATURES.md]: Unless the code itself is UwU-ified, keep the command names the same.

File: FEATURES.md [hunk 11]
```diff
@@ -1,52 +1,52 @@
...
 
-# Future features
+# Futuwe featuwes
 Things it doesn't have yet
-- Trapdoors
-- Sprint jumping in a 1x2 corridor
+- Twapdoows
+- Spwint jumping in a 1x2 cowwidow
 
-See <a href="https://github.com/cabaletta/baritone/issues">issues</a> for more.
```
Comment (Cerbiac) [inline FEATURES.md]: Broken Link

File: README.md [hunk 12]
```diff
@@ -1,86 +1,86 @@
-# Baritone
```
Comment (Cerbiac) [inline README.md]: A lot of broken links.

File: README.md [hunk 13]
```diff
@@ -1,86 +1,86 @@
...
+[ ^w^ [Awistois add-on integwation](https://img.shiewds.io/badge/Awistois%20add--on%20integwation-v1.3.4%20/%20v1.4.1-gween.svg)](https://gitwab.com/emc-mods-indwit/bawitonye_api)
+[ ^w^ [WWE integwation](https://img.shiewds.io/badge/WWE%20%22integwation%22-mastew%3F-gween.svg)](https://wwecwient.com/)
+[ ^w^ [Futuwe integwation](https://img.shiewds.io/badge/Futuwe%20integwation-Soon™%3F%3F%3F-wed.svg)](https://futuwecwient.nyet/)
+[ ^w^ [fowthebadge](https://fowthebadge.com/images/badges/buiwt-with-swag.svg)](http://fowthebadge.com/)
+[ ^w^ [fowthebadge](https://fowthebadge.com/images/badges/mom-made-pizza-wowws.svg)](http://fowthebadge.com/)
 
-A Minecraft pathfinder bot. 
+A Minyecwaft pathfindew bot. 
 
-Baritone is the pathfinding system used in [Impact](https://impactclient.net/) since 4.4. There's a [showcase video](https://www.youtube.com/watch?v=yI8hgW_m6dQ) made by @Adovin#3153 on Baritone's integration into Impact. [Here's](https://www.youtube.com/watch?v=StquF69-_wI) a video I made showing off what it can do.
```
Comment (Cerbiac) [inline README.md]: Broken Links

File: README.md [hunk 14]
```diff
@@ -1,86 +1,86 @@
...
+Bawitonye is the pathfinding system used in [Impact](https://impactcwient.nyet/) since 4.4. Thewe's a [showcase video](https://www.youtube.com/watch?v=yI8hgW_m6dQ) made by @Adovin#3153 on Bawitonye's integwation into Impact. [Hewe's](https://www.youtube.com/watch?v=StquF69-_wI) a video I made showing off what it can do.
 
-The easiest way to install Baritone is to install [Impact](https://impactclient.net/), which comes with Baritone. The second easiest way (for 1.12.2 only) is to install the v1.2.* forge api jar from [releases](https://github.com/cabaletta/baritone/releases). Otherwise, see [Installation & setup](SETUP.md). Once Baritone is installed, look [here](USAGE.md) for instructions on how to use it.
```
Comment (Cerbiac) [inline README.md]: Broken Link

File: README.md [hunk 15]
```diff
@@ -1,86 +1,86 @@
...
 
-This project is an updated version of [MineBot](https://github.com/leijurv/MineBot/),
-the original version of the bot for Minecraft 1.8.9, rebuilt for 1.12.2 and 1.13.2. Baritone focuses on reliability and particularly performance (it's over [30x faster](https://github.com/cabaletta/baritone/pull/180#issuecomment-423822928) than MineBot at calculating paths).
+This pwoject is an updated vewsion of [MinyeBot](https://github.com/weijuwv/MinyeBot/),
+the owiginyaw vewsion of the bot fow Minyecwaft 1.8.9, webuiwt fow 1.12.2 and 1.13.2. Bawitonye focuses on wewiabiwity and pawticuwawwy pewfowmance (it's uvw [30x fastew](https://github.com/cabawetta/bawitonye/puww/180#issuecomment-423822928) than MinyeBot at cawcuwating paths).
 
-Have committed at least once a day from Aug 1 2018 to Aug 1 2019.
+Have committed at weast once a day fwom Aug 1 2018 to Aug 1 2019.
 
-1Leijurv3DWTrGAfmmiTphjhXLvQiHg7K2
```
Comment (Cerbiac) [inline README.md]: False Commit Reference

File: README.md [hunk 16]
```diff
@@ -1,86 +1,86 @@
...
 
-The easiest way to install Baritone is to install [Impact](https://impactclient.net/), which comes with Baritone. The second easiest way (for 1.12.2 only) is to install the v1.2.* forge api jar from [releases](https://github.com/cabaletta/baritone/releases). Otherwise, see [Installation & setup](SETUP.md). Once Baritone is installed, look [here](USAGE.md) for instructions on how to use it.
+The easiest way to instaww Bawitonye is to instaww [Impact](https://impactcwient.nyet/), which comes with Bawitonye. The second easiest way (fow 1.12.2 onwy) is to instaww the v1.2.* fowge api jaw fwom [weweases](https://github.com/cabawetta/bawitonye/weweases). Othewwise, see [Instawwation & setup](SETUP.md). Once Bawitonye is instawwed, wook [hewe](USAGE.md) fow instwuctions on how to use it.
 
-For 1.14.4, [click here](https://www.dropbox.com/s/rkml3hjokd3qv0m/1.14.4-Baritone.zip?dl=1). Or [with optifine](https://github.com/cabaletta/baritone/issues/797).
```
Comment (Cerbiac) [inline README.md]: Broken Link

File: README.md [hunk 17]
```diff
@@ -1,86 +1,86 @@
...
+Fow 1.14.4, [cwick hewe](https://www.dwopbox.com/s/wkmw3hjokd3qv0m/1.14.4-Bawitonye.zip?dw=1). Ow [with optifinye](https://github.com/cabawetta/bawitonye/issues/797).
 
-This project is an updated version of [MineBot](https://github.com/leijurv/MineBot/),
```
Comment (Cerbiac) [inline README.md]: Broken Link

File: README.md [hunk 18]
```diff
@@ -1,86 +1,86 @@
...
-the original version of the bot for Minecraft 1.8.9, rebuilt for 1.12.2 and 1.13.2. Baritone focuses on reliability and particularly performance (it's over [30x faster](https://github.com/cabaletta/baritone/pull/180#issuecomment-423822928) than MineBot at calculating paths).
```
Comment (Cerbiac) [inline README.md]: Broken Link

File: README.md [hunk 19]
```diff
@@ -1,86 +1,86 @@
...
-Have committed at least once a day from Aug 1 2018 to Aug 1 2019.
+Have committed at weast once a day fwom Aug 1 2018 to Aug 1 2019.
 
-1Leijurv3DWTrGAfmmiTphjhXLvQiHg7K2
+1Weijuwv3DWTwGAfmmiTphjhXWvQiHg7K2
 
-# Getting Started
+# Getting Stawted
 
-Here are some links to help to get started:
```
Comment (Cerbiac) [inline README.md]: Broken Links

File: README.md [hunk 20]
```diff
@@ -1,86 +1,86 @@
...
 
-- [Settings](https://baritone.leijurv.com/baritone/api/Settings.html#field.detail)
+- [Settings](https://bawitonye.weijuwv.com/bawitonye/api/Settings.htmw#fiewd.detaiw)
 
-- [Usage (chat control)](USAGE.md)
+- [Usage (chat contwow)](USAGE.md)
 
 # API
 
-The API is heavily documented, you can find the Javadocs for the latest release [here](https://baritone.leijurv.com/).
```
Comment (Cerbiac) [inline README.md]: Broken Link

File: README.md [hunk 21]
```diff
@@ -1,86 +1,86 @@
...
-Please note that usage of anything located outside of the ``baritone.api`` package is not supported by the API release
```
Comment (Cerbiac) [inline README.md]: Unless the code is UwU-ified, Invalid File Name

File: README.md [hunk 22]
```diff
@@ -1,86 +1,86 @@
...
-jar.
+The API is heaviwy documented, you can find the Javadocs fow the watest wewease [hewe](https://bawitonye.weijuwv.com/).
+Pwease nyote that usage of anything wocated outside of the ``bawitonye.api`` package is nyot suppowted by the API wewease
+jaw.
 
-Below is an example of basic usage for changing some settings, and then pathing to a X/Z goal.
```
Comment (Cerbiac) [inline README.md]: Unless the code is UwU-ified, revert changes to code descriptions.

File: README.md [hunk 23]
```diff
@@ -1,86 +1,86 @@
...
-BaritoneAPI.getProvider().getPrimaryBaritone().getCustomGoalProcess().setGoalAndPath(new GoalXZ(10000, 20000));
+BawitonyeAPI.getPwovidew().getPwimawyBawitonye().getCustomGoawPwocess().setGoawAndPath(nyew GoawXZ(10000, 20000));
 ```
 
 # FAQ
 
-## Can I use Baritone as a library in my custom utility client?
+## Can I use Bawitonye as a wibwawy in my custom utiwity cwient?
 
-That's what it's for, sure! (As long as usage is in compliance with the LGPL 3.0 License)
```
Comment (Cerbiac) [inline README.md]: False Legal License

File: README.md [hunk 24]
```diff
@@ -1,86 +1,86 @@
...
+That's what it's fow, suwe ^w^  (As wong as usage is in compwiance with the WGPW 3.0 Wicense)
 
 ## How is it so fast?
 
-Magic. (Hours of [leijurv](https://github.com/leijurv/) enduring excruciating pain)
```
Comment (Cerbiac) [inline README.md]: Broken Link

File: SETUP.md [hunk 25]
```diff
@@ -1,106 +1,106 @@
-# Installation
+# Instawwation
 
-The easiest way to install Baritone is to install [Impact](https://impactclient.net/), which comes with Baritone.
```
Comment (Cerbiac) [inline SETUP.md]: Broken Link

File: SETUP.md [hunk 26]
```diff
@@ -1,106 +1,106 @@
...
+The easiest way to instaww Bawitonye is to instaww [Impact](https://impactcwient.nyet/), which comes with Bawitonye.
 
-For 1.14.4, [click here](https://www.dropbox.com/s/rkml3hjokd3qv0m/1.14.4-Baritone.zip?dl=1).
```
Comment (Cerbiac) [inline SETUP.md]: Broken Link

File: SETUP.md [hunk 27]
```diff
@@ -1,106 +1,106 @@
...
 
-Once Baritone is installed, look [here](USAGE.md) for instructions on how to use it.
+Once Bawitonye is instawwed, wook [hewe](USAGE.md) fow instwuctions on how to use it.
 
-## Prebuilt official releases
-These releases are not always completely up to date with latest features, and are only released from `master`. (so if you want `backfill-2` branch for example, you'll have to build it yourself)
+## Pwebuiwt officiaw weweases
+These weweases awe nyot awways compwetewy up to date with watest featuwes, and awe onwy weweased fwom `mastew`. (so if you want `backfiww-2` bwanch fow exampwe, you'ww have to buiwd it youwsewf)
 
-Link to the releases page: [Releases](https://github.com/cabaletta/baritone/releases)
```
Comment (Cerbiac) [inline SETUP.md]: Broken Link

File: SETUP.md [hunk 28]
```diff
@@ -1,106 +1,106 @@
...
+Wink to the weweases page: [Weweases](https://github.com/cabawetta/bawitonye/weweases)
 
-v1.2.* is for 1.12.2, v1.3.* is for 1.13.2
+v1.2.* is fow 1.12.2, v1.3.* is fow 1.13.2
 
-Any official release will be GPG signed by leijurv (44A3EA646EADAC6A) and ZeroMemes (73A788379A197567). Please verify that the hash of the file you download is in `checksums.txt` and that `checksums_signed.asc` is a valid signature by those two public keys of `checksums.txt`.
```
Comment (Cerbiac) [inline SETUP.md]: False Legalities | False File Name

File: SETUP.md [hunk 29]
```diff
@@ -1,106 +1,106 @@
...
-Link to the releases page: [Releases](https://github.com/cabaletta/baritone/releases)
+Wink to the weweases page: [Weweases](https://github.com/cabawetta/bawitonye/weweases)
 
-v1.2.* is for 1.12.2, v1.3.* is for 1.13.2
+v1.2.* is fow 1.12.2, v1.3.* is fow 1.13.2
 
-Any official release will be GPG signed by leijurv (44A3EA646EADAC6A) and ZeroMemes (73A788379A197567). Please verify that the hash of the file you download is in `checksums.txt` and that `checksums_signed.asc` is a valid signature by those two public keys of `checksums.txt`. 
+Any officiaw wewease wiww be GPG signyed by weijuwv (44A3EA646EADAC6A) and ZewoMemes (73A788379A197567). Pwease vewify that the hash of the fiwe you downwoad is in `checksums.txt` and that `checksums_signyed.asc` is a vawid signyatuwe by those two pubwic keys of `checksums.txt`. 
 
-The build is fully deterministic and reproducible, and you can verify Travis did it properly by running `docker build --no-cache -t cabaletta/baritone .` yourself and comparing the shasum. This works identically on Travis, Mac, and Linux (if you have docker on Windows, I'd be grateful if you could let me know if it works there too).
```
Comment (Cerbiac) [inline SETUP.md]: Code

File: SETUP.md [hunk 30]
```diff
@@ -1,106 +1,106 @@
...
+The buiwd is fuwwy detewminyistic and wepwoducibwe, and you can vewify Twavis did it pwopewwy by wunnying `dockew buiwd --nyo-cache -t cabawetta/bawitonye .` youwsewf and compawing the shasum. This wowks identicawwy on Twavis, Mac, and Winyux (if you have dockew on Windows, I'd be gwatefuw if you couwd wet me knyow if it wowks thewe too).
 
 
-## Artifacts
+## Awtifacts
 
-Building Baritone will result in 5 artifacts created in the ``dist`` directory. These are the same as the artifacts created in the [releases](https://github.com/cabaletta/baritone/releases).
```
Comment (Cerbiac) [inline SETUP.md]: Link

File: SETUP.md [hunk 31]
```diff
@@ -1,106 +1,106 @@
...
+Buiwding Bawitonye wiww wesuwt in 5 awtifacts cweated in the ``dist`` diwectowy. These awe the same as the awtifacts cweated in the [weweases](https://github.com/cabawetta/bawitonye/weweases).
 
-**The Forge release can simply be added as a Forge mod.**
+**The Fowge wewease can simpwy be added as a Fowge mod.**
 
-If another one of your Forge mods has a Baritone integration, you want `baritone-api-forge-VERSION.jar`. Otherwise, you want `baritone-standalone-forge-VERSION.jar`
```
Comment (Cerbiac) [inline SETUP.md]: File Names

File: SETUP.md [hunk 32]
```diff
@@ -1,106 +1,106 @@
...
+## Mowe Info
+To wepwace out Impact 4.5's Bawitonye buiwd with a customized onye, buiwd Bawitonye as abuv then copy & **wenyame** `dist/bawitonye-api-$VEWSION$.jaw` into `minyecwaft/wibwawies/cabawetta/bawitonye-api/1.2/bawitonye-api-1.2.jaw`, wepwacing the jaw that was pweviouswy thewe. You awso nyeed to edit `minyecwaft/vewsions/1.12.2-Impact_4.5/1.12.2-Impact_4.5.json`, find the winye `"nyame": "cabawetta:bawitonye-api:1.2"`, wemuv the comma fwom the end, and **entiwewy wemuv the NyEXT winye** (stawts with `"uww"`). **Westawt youw waunchew** then woad as nyowmaw. 
 
-You can verify whether or not it worked by running `.b version` in chat (only valid in Impact). It should print out the version that you downloaded. Note: The version that comes with 4.5 is `v1.2.3`.
```
Comment (Cerbiac) [inline SETUP.md]: Command

File: SETUP.md [hunk 33]
```diff
@@ -1,106 +1,106 @@
...
+You can vewify whethew ow nyot it wowked by wunnying `.b vewsion` in chat (onwy vawid in Impact). It shouwd pwint out the vewsion that you downwoaded. Nyote: The vewsion that comes with 4.5 is `v1.2.3`.
 
-## Build it yourself
-- Clone or download Baritone
+## Buiwd it youwsewf
+- Cwonye ow downwoad Bawitonye
 
-  ![Image](https://i.imgur.com/kbqBtoN.png)
```
Comment (Cerbiac) [inline SETUP.md]: Link

File: SETUP.md [hunk 34]
```diff
@@ -1,106 +1,106 @@
...
-  - If you choose to download, make sure you extract the ZIP archive.
-- Follow one of the instruction sets below, based on your preference
+   ^w^ [Image](https://i.imguw.com/kbqBtoN.png)
+  - If you choose to downwoad, make suwe you extwact the ZIP awchive.
+- Fowwow onye of the instwuction sets bewow, based on youw pwefewence
 
-## Command Line
-On Mac OSX and Linux, use `./gradlew` instead of `gradlew`.
```
Comment (Cerbiac) [inline SETUP.md]: File Name

File: SETUP.md [hunk 35]
```diff
@@ -1,106 +1,106 @@
...
+## Command Winye
+On Mac OSX and Winyux, use `./gwadwew` instead of `gwadwew`.
 
-Setting up the Environment:
+Setting up the Enviwonment:
 
 ```
-$ gradlew setupDecompWorkspace
```
Comment (Cerbiac) [inline SETUP.md]: Code

File: SETUP.md [hunk 36]
```diff
@@ -1,106 +1,106 @@
...
-$ gradlew --refresh-dependencies
```
Comment (Cerbiac) [inline SETUP.md]: Code

File: SETUP.md [hunk 37]
```diff
@@ -1,106 +1,106 @@
...
+$ gwadwew setupDecompWowkspace
+$ gwadwew --wefwesh-dependencies
 ```
 
-Running Baritone:
+Wunnying Bawitonye:
 
 ```
-$ gradlew runClient
```
Comment (Cerbiac) [inline SETUP.md]: Code

File: SETUP.md [hunk 38]
```diff
@@ -1,106 +1,106 @@
...
+$ gwadwew wunCwient
 ```
 
-For information on how to build baritone, see [Building Baritone](#building-baritone)
```
Comment (Cerbiac) [inline SETUP.md]: Link

File: SETUP.md [hunk 39]
```diff
@@ -1,106 +1,106 @@
...
+Fow infowmation on how to buiwd bawitonye, see [Buiwding Bawitonye](#buiwding-bawitonye)
 
-## IntelliJ
-- Open the project in IntelliJ as a Gradle project
```
Comment (Cerbiac) [inline SETUP.md]: False Image Links | False Code/Commands

File: SETUP.md [hunk 40]
```diff
@@ -1,106 +1,106 @@
...
+# Buiwding
 
-Make sure that you have properly [setup](#setup) the environment before trying to build it.
+Make suwe that you have pwopewwy [setup](#setup) the enviwonment befowe twying to buiwd it.
 
-## Command Line
+## Command Winye
 
 ```
-$ gradlew build
```
Comment (Cerbiac) [inline SETUP.md]: Code

File: SETUP.md [hunk 41]
```diff
@@ -1,106 +1,106 @@
...
+$ gwadwew buiwd
 ```
 
-## IntelliJ
+## IntewwiJ
 
-- Navigate to the gradle tasks on the right tab as follows
+- Nyavigate to the gwadwe tasks on the wight tab as fowwows
 
-  ![Image](https://i.imgur.com/PE6r9iN.png)
```
Comment (Cerbiac) [inline SETUP.md]: Link

File: SETUP.md [hunk 42]
```diff
@@ -1,106 +1,106 @@
...
+   ^w^ [Image](https://i.imguw.com/PE6w9iN.png)
 
-- Double click on **build** to run it
```
Comment (Cerbiac) [inline SETUP.md]: Code

File: CODE_OF_CONDUCT.md (hunk 3 above)
Comment (PF94) [reply CODE_OF_CONDUCT.md]: a lot of broken links lol

**Comment by S-B99** [comment]
@olibols just curious, what did you use to change the letters to uwu? I just need the pattern for something.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (TheK0tYaRa) [reply CODE_OF_CONDUCT.md:80]: > Broken Link

Now it's not

**Comment by flae7** [comment]
Broken link

**Comment by zPeanut** [comment]
what the fuck

**Comment by iBuyMountainDew** [comment]
why is this a thing

**Comment by dominikaaaa** [comment]
uwu :3

**Comment by SomethingGeneric** [comment]
What is happening

**Comment by Colonizor48** [comment]
Sentient life was a mistake. Reject humanity. Return to eubacteria.

**Comment by 0-x-2-2** [comment]
vouch

**Comment by EthanHindmarsh** [comment]
major improvement

**Comment by feliscatusmeows** [comment]
pls merge or bad

**Comment by StijnSimons** [comment]
le epic funny from 2019 still getting comments 😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂

**Review by zPeanut — APPROVED** [review]
LGTM!

**Comment by AstroOrbis** [comment]
idea: make a new branch called OwO and make EVERYTHING OwO over there
Commands, variable names, everything

**Comment by leijurv** [comment]
that would be funny i think

**Comment by joerez** [comment]
@leijurv image

**Comment by zPeanut** [comment]
> that would be funny i think

NO

**Comment by Cerbiac** [comment]
Imagine being dead since 2019 ♿

**Comment by StijnSimons** [comment]
when the unfunny

**Comment by AstroOrbis** [comment]
I'll do it
Watch me

**Comment by Ayfri** [comment]
You should resolve the conflicts

**Comment by Whanos** [comment]
how come this hasn't been pushed yet

**Comment by StijnSimons** [comment]
i stg stop commenting on this

**Comment by maska228** [comment]
> i stg stop commenting on this

no u

**Comment by maska228** [comment]
> i stg stop commenting on this

@StijnSimons no u

**Comment by StijnSimons** [comment]
XDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDdddd

**Comment by zPeanut** [comment]
stop spamming my emails

**Comment by olibols** [comment]
Lots of changes since I took on this PR as lead developer. I'll start working soon.

**Comment by bddvlpr** [comment]
death

**Pull request closed without merging**